packaging/deb/ogon-channels/control

```diff
@@ -0,0 +1,56 @@
+Source: ogon-channels
+Maintainer: ogon packaging team <ogon+package@thincast.com>
+Section: net
+Priority: optional
+Standards-Version: 3.9.7
+Homepage: http://ogon-project.com/
+Vcs-Browser: https://github.com/ogon/ogon
+Vcs-Git: https://github.com/ogon/ogon
```

Comment:
these urls aren't pointing to the right place

Reply:
@hardening oops ^^ fixed.